acme_tiny.py

```diff
@@ -63,12 +63,12 @@ def _send_signed_request(url, payload, err_msg, depth=0):
 
     # helper function - poll until complete
     def _poll_until_not(url, pending_statuses, err_msg):
-        while True:
+        result, t0 = None, time.time()
+        while result is None or result['status'] in pending_statuses:
+            assert (time.time() - t0 < 3600), "Polling timeout" # 1 hour timeout
+            time.sleep(0 if result is None else 2)
             result, _, _ = _do_request(url, err_msg=err_msg)
```

Comment:
This one needs to be POST-as-GET (not the one above).

Reply:
Oh, ha. Yes, I originally did this in the original fix (https://github.com/diafygi/acme-tiny/pull/227/commits/1b6c6bc0e67aa28fac259c3077883594d3c871d5#diff-a3795abce46ab9122de22623f698879aR67), but looks like I forgot to update the timeout fix.

Reply:
Should now be fixed in diafygi/acme-tiny@cb094cf